tests/integration/postgresql/postgresql_spec.rb

```diff
@@ -138,6 +138,11 @@
     its('output') { should match /example_role/ }
   end
 
+  # Assert that connection limit is reflected correctly
+    describe sql.query("SELECT rolconnlimit FROM pg_roles WHERE rolname='example_role'") do
+    its('output') { should eq('100') }
+  end
```

Comment:
Can you add a test that a user that's not in the pillar will stay at `-1`? I think this is a good check to ensure upcoming changes might not mess with all users, etc.

Reply:
good idea, updated tests 👍